NOTICE

```diff
@@ -1,4 +1,4 @@
-clj-symphony
-Copyright 2016 Fintech Open Source Foundation
+clj-symphony - FINOS
+Copyright 2016-2019 Symphony LLC
```

Comment:
This copyright notice is incorrect.  [Copyright notices only include the year of first publication](https://en.wikipedia.org/wiki/Copyright_notice#Form_of_notice_for_visually_perceptible_copies) - they are inherently open ended from that year forward.

Reply:
Furthermore, this code was developed by FINOS staff (i.e. myself), so the original copyright attribution is correct.  Symphony LLC has had no input to this project whatsoever.